ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -169,13 +169,19 @@ public void processResponse(RestRequest restRequest, RestResponseChannel respons
                     responseChannel.setStatus(ResponseStatus.PartialContent);
                   }
                   setGetBlobResponseHeaders(blobInfo, options, responseChannel);
+                  setBlobPropertiesHeaders(blobInfo.getBlobProperties(), responseChannel);
+                  setAccountAndContainerHeaders(restRequest, responseChannel);
+                  setUserMetadataHeaders(blobInfo.getUserMetadata(), responseChannel);
```

Comment:
hmm i wonder if there's a good way to send old style user metadata on getBlob responses. Maybe we could make a special header for it and base64 encode the data.

Reply:
Yeah we can consider that in the future - however header sizes have limits so that might need some thinking